packages/sanity/src/core/changeIndicators/ElementWithChangeBar.tsx

```diff
@@ -35,34 +30,21 @@ export function ElementWithChangeBar(props: {
             aria-label="Review changes"
             data-testid="change-bar__button"
             onClick={isReviewChangesOpen ? undefined : onOpenReviewChanges}
-            onMouseEnter={handleMouseEnter}
-            onMouseLeave={handleMouseLeave}
```

Comment:
Similarly, this internal state has been removed as `ChangeBarWrapper` isn't varying on the value of `hover`